docs/interactions/structured-messages/list.md

```diff
@@ -40,13 +40,19 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 | **`in_reply_to_id`** | String | ID of the message being replied to. |
 | **`structured_content`** | Object | Payload of the structured message. |
 | **Structured Content Settings** | | |
-| **`structured_content.type`** | String | Type of the structured message. Must be "select". |
+| **`structured_content.type`** | String | Type of the structured message. Must be "list". |
 | **`structured_content.title`** | String | Title of the list structured message. Limited to 1024 characters. |
 | **`structured_content.subtitle`** | String | **Optional**. Subtitle of the list structured message. Limited to 512 characters. |
-| **`structured_content.items`** | Array | An array of items representing the options presented to the customer. A maximum of 13 items is supported. |
+| **`structured_content.sections`** | Array | **Optional**. An array of sections in which the items will be organized.<br>Limited to 10 elements.<br>If blank, every item will be part of the same section. |
+| **Section Settings** | | |
+| **`structured_content.sections.title`** | String | **Optional if there's only a single section**. The title of the section.<br>Limited to 24 characters. |
```

Comment:
I see in documentation that we say : 
```
Optional if there is no section
```
Which isn't exactly the same as only a single section no ?

Reply:
The section title is optional if there is a single section (ofc it's optional if there is no section, I mean no section = no title anyway)